app/src/forms/form/exportService.js

```diff
@@ -272,6 +281,24 @@ const service = {
     const result = await service._formatData(exportFormat, exportType,exportTemplate, form, data, columns, version);
 
     return { data: result.data, headers: result.headers };
+  },
+  _getListSubmittersByFormId :  (formId, obj ) => {
```

Comment:
Shouldn't this be an async function since the knex query builder is an async function that can be awaited?